ext/opentelemetry-ext-opentracing-shim/src/opentelemetry/ext/opentracingshim/__init__.py

```diff
@@ -0,0 +1,232 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from contextlib import contextmanager
+
+import opentracing
+
+from opentelemetry.ext.opentracingshim import util
+from opentelemetry.trace import Event
+
+
+def create_tracer(tracer, scope_manager=None):
+    return TracerWrapper(tracer, scope_manager)
+
+
+class SpanContextWrapper(opentracing.SpanContext):
+    def __init__(self, otel_context):
+        self._otel_context = otel_context
+
+    def unwrap(self):
+        """Returns the wrapped OpenTelemetry `SpanContext` object."""
+
+        return self._otel_context
+
+    @property
+    def baggage(self):
+        raise NotImplementedError
+        # TODO: Implement.
+
+
+class SpanWrapper(opentracing.Span):
+    def __init__(self, tracer, context, span):
```

Comment:
So `tracer` and `context` are the wrapped types, but `span` is a regular `opentelemetry.trace.Span`?  This makes sense, but might be confusing if you're expecting to use `SpanWrapper` as a drop-in replacement for `opentelemetry.Span`.

So the constructor isn't part of the OpenTracing API?

Reply:
The idea IHMO should be to only allow users to use `create_tracer()`, and don't allow them to play with the underlying pieces (as this will also make things overall simpler).

Reply:
>The idea IHMO should be to only allow users to use create_tracer()

Yeah, I didn't have users directly creating `SpanWrapper` objects in mind. But anyway @c24t, I guess this will be better when I add type info and docstrings (right?).

Reply:
Yup, that SGTM.